### PR TITLE
303 success section

### LINKED ISSUE #303: [FRONTEND] Create success section
**Is your feature request related to a problem? Please describe.**

Image

**BEFORE MERGING**

- [x] Storybooks created where possible
- [ ] Tested with [nock](https://tanstack.com/query/v3/docs/framework/react/guides/testing) if data fetching
- [ ] Tests written for critical interactions
- [ ] PR Reviewed (For non-trivial changes)
- [ ] Changes tested after rebasing on master or merging in master (_hint_: `git fetch origin master:master`, then `git rebase master` or `git merge master`)
- [ ] All required PR checks passing

### PR DESCRIPTION
**Pull request by zlrkw11**
closes #303

### CODE REVIEW

**Comment by zlrkw11** [comment]
image

File: client/src/components/composite/SignUpForm/utils/Utils.tsx [hunk 1]
```diff
@@ -44,6 +45,9 @@ export const useCurrentStep = (): PAGES => {
     case ACCOUNT_SETUP_ROUTE:
       return PAGES.AccountSetup
 
+    case SUCCESS_ROUTE:
```
Comment (bcho892) [inline client/src/components/composite/SignUpForm/utils/Utils.tsx:48]: Had to add this

File: client/src/components/composite/SignUpForm/SignUpForm.tsx [hunk 2]
```diff
@@ -59,6 +59,8 @@ export const ProtectedSignUpForm = ({
         return <Navigate to={oneLevelUp(PERSONAL_ROUTE_1)} replace />
       }
       break
+    case PAGES.Success:
```
Comment (bcho892) [inline client/src/components/composite/SignUpForm/SignUpForm.tsx:62]: and this

**Review by bcho892 — APPROVED** [review]
We need to make sure that these changes are tested with the webpage was up and running. I spotted a pretty big bug but it would have been prevented with just checking out the dev page.

**Pull request merged**